tests/kafkatest/tests/consumer_rolling_upgrade.py

```diff
@@ -15,32 +15,23 @@
 
 from ducktape.utils.util import wait_until
 
-from kafkatest.tests.kafka_test import KafkaTest
-from kafkatest.services.verifiable_consumer import VerifiableConsumer
+from kafkatest.tests.verifiable_consumer_test import VerifiableConsumerTest
 from kafkatest.services.kafka import TopicPartition
 
-class ConsumerRollingUpgradeTest(KafkaTest):
+class ConsumerRollingUpgradeTest(VerifiableConsumerTest):
     TOPIC = "test_topic"
     NUM_PARTITIONS = 4
-    GROUP_ID = "test_group_id"
     RANGE = "org.apache.kafka.clients.consumer.RangeAssignor"
     ROUND_ROBIN = "org.apache.kafka.clients.consumer.RoundRobinAssignor"
 
     def __init__(self, test_context):
-        super(ConsumerRollingUpgradeTest, self).__init__(test_context, num_zk=1, num_brokers=1, topics={
+        super(ConsumerRollingUpgradeTest, self).__init__(test_context, num_consumers=2, num_producers=0,
+                                                         num_zk=1, num_brokers=1, topics={
             self.TOPIC : { 'partitions': self.NUM_PARTITIONS, 'replication-factor': 1 }
         })
         self.num_consumers = 2
         self.session_timeout = 10000
```

Comment:
If I'm reading the rest of the patch right, it looks like `session_timeout` in these tests has generally been removed. It looks to me like this is also just leftover and can now be removed (and should be since it uses ms whereas all the new stuff uses seconds)?